### PR TITLE
NIFI-9809 Disable Kudu tests on arm64

### PR DESCRIPTION
**Pull request by kevdoran**
This PR disables kudu tests on arm64 platforms, as they require the `kudu-binary` artifact that is not available on arm64 platforms (at least not yet, the Kudu community, from conversations I've had, is investigating it).

With this PR, I'm happy to share that all build issues on arm64 are resolved, at least for M1 Macs where I've been testing.

There are still some runtime issues to address before we can claim full arm64 compatibility, but by disabling tests, this can be built on an M1 Mac. E.g.:

```
 mvn -T2C clean install -DskipTests
```

# Summary

[NIFI-9809](https://issues.apache.org/jira/browse/NIFI-9809)

# Tracking

Please complete the following tracking steps prior to pull request creation.

### Issue Tracking

- [ ] [Apache NiFi Jira](https://issues.apache.org/jira/browse/NIFI) issue created

### Pull Request Tracking

- [ ] Pull Request title starts with Apache NiFi Jira issue number, such as `NIFI-00000`
- [ ] Pull Request commit message starts with Apache NiFi Jira issue number, as such `NIFI-00000`

### Pull Request Formatting

- [ ] Pull Request based on current revision of the `main` branch
- [ ] Pull Request refers to a feature branch with one commit containing changes

# Verification

Please indicate the verification steps performed prior to pull request creation.

### Build

- [ ] Build completed using `mvn clean install -P contrib-check`
  - [ ] JDK 8
  - [ ] JDK 11
  - [ ] JDK 17

### Licensing

- [ ] New dependencies are compatible with the [Apache License 2.0](https://apache.org/licenses/LICENSE-2.0) according to the [License Policy](https://www.apache.org/legal/resolved.html)
- [ ] New dependencies are documented in applicable `LICENSE` and `NOTICE` files

### Documentation

- [ ] Documentation formatting appears as expected in rendered files

### CODE REVIEW

**Comment by ottobackwards** [comment]
LGTM! +1

**Review by exceptionfactory — COMMENTED** [review]
Thanks for the improvement @kevdoran! The changes look good, but can the profile definition be centralized in the parent `nifi-kudu-bundle`? Properties from a parent module should apply to child modules, so centralizing the settings in the parent module would reduce duplication.

**Comment by kevdoran** [comment]
Oh, good call. I didn't consider that, but I like consolidating into the parent. Let me give that a try and I'll push an update

File: nifi-nar-bundles/nifi-kudu-bundle/nifi-kudu-processors/pom.xml [hunk 1]
```diff
@@ -116,22 +116,11 @@
...
-                <exclude.tests>**/*.java</exclude.tests>
-            </properties>
-        </profile>
-        <profile>
-            <id>kudu-linux</id>
+            <id>kudu-linux-x86</id>
             <activation>
                 <os>
                     <family>Unix</family>
+                    <arch>x86-64</arch>
```
Comment (exceptionfactory) [inline nifi-nar-bundles/nifi-kudu-bundle/nifi-kudu-processors/pom.xml:123]: This architecture setting excludes running on `amd64`.

**Review by exceptionfactory — CHANGES_REQUESTED** [review]
Prior to merging, observed that using `x86_64` excludes `amd64` Linux-based builds, so this should be corrected.

**Review by exceptionfactory — APPROVED** [review]
Thanks for making the adjustments @kevdoran, the current version looks good! +1 merging

**Pull request closed without merging**